spec/models/pipeline_run_spec.rb

```diff
@@ -158,4 +183,68 @@
       end
     end
   end
+  context "ensure that loading amr counts works properly" do
```

Comment:
Do these tests trigger the behavior you were seeing with the annotation string and the ';'s?

Reply:
Nah cause I took out the regex

Reply:
But they should rigger that use case, to make sure it is fixed...

Reply:
Yeah it's in there!